src/content/todoist.js

```diff
@@ -255,3 +265,22 @@ function getParentIfProject (elem) {
 
   return project;
 }
+
+function separateAndCheck (str, projects) {
+  const words = str.split('/');
+  if (words.length < 2) {
+    return str;
+  }
+  const project = words[0].trim();
+  const section = words[1].trim();
+  if (isExist(project, projects)) {
+    return project;
+  } else if (isExist(section, projects)) {
+    return section;
+  }
+  return str;
+}
+
+function isExist(str, arr) {
```

Comment:
```suggestion
function contains(str, arr) {
```

Better naming. and should swap arguments too: `contains(arr, str)`

or use native `Array.includes`:
- https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes